package.json

```diff
@@ -293,6 +290,7 @@
 		"@types/wordpress__editor": "npm:noop-package@1.0.0",
 		"@types/wordpress__notices": "npm:noop-package@1.0.0",
 		"@types/wordpress__rich-text": "npm:noop-package@1.0.0",
+		"@types/wordpress__data-controls/@wordpress/data": "npm:noop-package@1.0.0",
```

Comment:
This is ok because `@wordpress/data` is only a dev dependency of the type package. Unfortunately DT doesn't allow dev dependencies, and using `@wordpress/data` for the test file is important. However, the dependency doesn't impact the actual type definitions.

Reply:
Sounds good. I'd love to see all those hacks removed once all packages expose their own types.